apps/researcher/src/messages/nl/messages.json

```diff
@@ -153,54 +153,54 @@
+  "attributionIncompleteLabel": "Collecties en items in onze instelling hebben een onvolledige, onjuiste en/of ontbrekende toeschrijving en beschrijving. We gebruiken deze _notice_ om dit materiaal duidelijk te identificeren zodat het kan worden bijgewerkt of gecorrigeerd door gemeenschappen van herkomst. Onze instelling zet zich in voor samenwerking en partnerschappen om dit probleem van onjuiste of ontbrekende toeschrijving aan te pakken.",
   "authorization": "Autorisatie",
-  "authorizationLabel": "De Autorisatiekennisgeving wordt gebruikt om aan te geven dat er door middel van overleg distincte tribale of gemeenschapsautorisatie moet worden verkregen voor elke reproductie van dit item, inclusief het gebruik in tentoonstellingen en publicaties in print of online.",
+  "authorizationLabel": "De _Autorisatie-notice_ wordt gebruikt om aan te geven dat er door middel van overleg, specifieke gemeenschapsautorisatie moet worden verkregen voor elke reproductie van dit item, inclusief het gebruik in tentoonstellingen en publicaties in print of online.",
   "belonging": "Toebehoren",
-  "belongingLabel": "De Toebehorenkennisgeving wordt gebruikt om aan te geven dat alleen specifieke personen die tot gemeenschappen, stammen, clans en verenigingen behoren, het recht hebben om deze specifieke items of collecties te bekijken en te behandelen.",
+  "belongingLabel": "De _Toebehoren-notice_ wordt gebruikt om aan te geven dat alleen specifieke personen die tot gemeenschappen, stammen, clans en verenigingen behoren, het recht hebben om deze specifieke items of collecties te bekijken en te behandelen.",
   "caring": "Verzorging",
-  "caringLabel": "De Verzorgingskennisgeving wordt gebruikt om aan te geven dat gespecialiseerde culturele zorg vereist is. Zorg voor items of collecties vereist specifieke gemeenschapsrichtlijnen voor opslag en handling.",
+  "caringLabel": "De _Verzorgings-notice_ wordt gebruikt om aan te geven dat gespecialiseerde culturele zorg vereist is. Zorg voor items of collecties vereist specifieke gemeenschapsrichtlijnen voor opslag en hantering.",
   "genderAware": "Genderbewust",
-  "genderAwareLabel": "De Genderbewust Kennisgeving wordt gebruikt om aan te geven dat er specifieke genderverantwoordelijkheden zijn die van invloed zijn op wie toegang kan krijgen tot, kijken naar en/of omgaan met dit item of deze collectie.",
-  "leaveUndisturbed": "Ongestoord Laten",
-  "leaveUndisturbedLabel": "De Ongestoord Laten Kennisgeving wordt gebruikt om aan te geven dat items in een collectie niet mogen worden verstoord. Ze mogen alleen onder zeer beperkte omstandigheden volgens gemeenschapsrichtlijnen worden verplaatst of verstoord.",
+  "genderAwareLabel": "De _Genderbewust-notice_ wordt gebruikt om aan te geven dat er specifieke genderverantwoordelijkheden zijn die van invloed zijn op wie toegang kan krijgen tot, kijken naar en/of omgaan met dit item of deze collectie.",
+  "leaveUndisturbed": "Onverstoord Laten",
```

Comment:
Onverstoord klinkt vreemd, iemand is wel onverstoorbaar, maar een object onverstoord laten, voelt vreemd. Alternatief: 'Ongemoeid laten', 'Met rust laten'?

Reply:
'Ongemoeid laten' is okay, wijzig ik het